iio/iio_axi_adc/iio_axi_adc.h

```diff
@@ -53,11 +53,14 @@
 
 /**
  * @struct iio_axi_adc_desc
- * @brief iio desciptor.
+ * @brief iio_axi_adc_descriptor
  */
 struct iio_axi_adc_desc {
```

Comment:
This refactoring iio_axi_adc -> iio_axi_adc_desc likely breaks existing projects, we can drop it.